src/scripts/main.js

```diff
@@ -1 +1,43 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(() => {
+    reject(new Error('First promise was rejected'));
+  }, 3000);
+});
+const secondPromise = new Promise(resolve => {
```

Comment:
Really hard to read. Add spaces where needed, please
```suggestion
});

const secondPromise = new Promise(resolve => {
```